src/Component/page/Footer/Footer.css

```diff
@@ -59,12 +67,13 @@
    padding: 2rem;
 }
 .footer-text{
-    margin-left:rem;  
+    margin-left:5rem;  
 }
 
    }
 
-   @media (max-width: 767px){
+   /* Media Queries for mobile  */
+   @media (min-width:320px) and (max-width: 480px) {
```

Comment:
Shouldn't have a min-width here - this should apply to everything below 480px, with this code, things below 320px will have none of these styles applied